File: psp/card-service/src/main/java/com/psp/cardservice/dto/DataServicePaymentResponse.java
package com.psp.cardservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DataServicePaymentResponse {
    private String id;
    private String merchantId;
    private String merchantPassword;
    private Double amount;
    private String currency;
    private String merchantOrderId;
    private Long merchantTimestamp;
}

File: psp/card-service-csharp/CardService.Domain/DTOs/DataServicePaymentResponse.cs
using CardService.Domain.Enums;

namespace CardService.Domain.DTOs
{
    public record DataServicePaymentResponse(
        Guid Id,
        string MerchantId,
        string MerchantPassword,
        double Amount,
        Currency Currency,
        string MerchantOrderId,
        DateTime MerchantTimestamp
    );
}
